flight/Modules/UAVOMSPBridge/UAVOMSPBridge.c

```diff
@@ -283,12 +284,13 @@ static void _msp_send_analog(struct msp_bridge *m)
 		FlightBatterySettingsGet(&batSettings);
 	}
 
-	// TODO:  Verify these with a board that has working power stuff
 	if (batSettings.VoltagePin != FLIGHTBATTERYSETTINGS_VOLTAGEPIN_NONE)
 		data.status.vbat = (uint8_t)lroundf(batState.Voltage * 10);
 
-	if (batSettings.CurrentPin != FLIGHTBATTERYSETTINGS_CURRENTPIN_NONE)
-		data.status.current = lroundf(batState.Current * 10);
+	if (batSettings.CurrentPin != FLIGHTBATTERYSETTINGS_CURRENTPIN_NONE) {
+		data.status.current = lroundf(batState.Current * 100);
```

Comment:
> ... does it mean this variable contains a value equivalent to 100 A in whatever scaling is being used, or current multiplied by 100, or current divided by 100, or...?

That's a fair question and I don't have a way to answer it. What's clear to me, though, is that it's *not* just current and should be investigated more before making assumptions. 

I especially see this kind of scaling error with degrees and radians. Moving to this kind of naming has saved my team tons of time.